### DISCUSSION TITLE
Let's talk about html.org.in website

### DISCUSSION THREAD

**Discussion by Diptenusarkar** [html-org-in/html.org.in#104] [Ideas]
Hello @abulgit  Can you work with me? For this project.
Let's make this website more accessible to all.

**Comment by abulgit** [comment]
Yeah , Sure

**Comment by abulgit** [comment]
### I have a few ideas to improve this website:

**1. The UI could be more interactive and responsive.
2. Actual video tutorials for learners should be provided.
3. Unique content and teaching methods should be incorporated.
4. The website should aim to provide everything a learner needs to know about HTML,
      making it a one-stop solution for learning HTML.
6. Also it's good to have a backend as well.**

**Comment by Diptenusarkar** [comment]
Exactly, Let's do it. 
What if, we write our ideas and then arrange a meeting? 
I think this will help to decide our roadmap. 
What did you think?

**Comment by abulgit** [comment]
Yeah Definitely.

**Comment by Diptenusarkar** [comment]
Hello @abulgit , 
Let's team up to create an outstanding website! Are you ready to start working together?

**Comment by abulgit** [comment]
I have my sem exam, I will look into it after that.

**Reply by Diptenusarkar** [reply]
Hello @abulgit
